File: Python/Functions/function.py



# def greet(name):
#     print(f"Hello {name}")

# greet("John")

# def sum(a, b):
#     return a + b

# print(sum(3, 4))

def multiple_items(*args):
    print(args)
    print(type(args))

multiple_items(1,2,3,4,5)

def mult_named_items(**kwargs):
    print(kwargs)
    print(type(kwargs))

mult_named_items(name="John", age=25, city="New York")

File: C++/Functions/function.cpp
#include <iostream>

void greet(){
    std::cout << "Good Morning!" << std::endl;
};


void sum(int a, int b){
    std::cout << (a + b) << std::endl;
};

void mult(int a = 5, int b = 2){
    std::cout << (a * b) << std::endl;
};

int main(){
    greet();

    sum(2, 3);

    mult(2, 2);
};